exetera/core/fields.py

```diff
@@ -116,6 +125,12 @@ def _ensure_valid(self):
 class MemoryField(Field):
 
     def __init__(self, session):
+        """
+        Construct a field stored in memory only, often used when perform arithmetic/comparison operations from storage
```

Comment:
Constructor for memory-based Fields. This initializer should only be called by Field types that are subclasses of MemoryField, and never directly by users of ExeTera.